modules/oadp-backing-up-pvs-csi.adoc

```diff
@@ -27,7 +27,8 @@ metadata:
   labels:
```

Comment:
Vsclass example is missing the annotation. 

  annotations:
    snapshot.storage.kubernetes.io/is-default-class: true <2>

Reply:
@PrasadJoshi12 
I added it to the example. Could you check that I did it correctly?